tools/pepquery/pepquery.xml

```diff
@@ -0,0 +1,196 @@
+                    <param label="Annotation Files" name="anno" type="data_collection" argument="-anno" help="Annotation files for the VCF file. Please follow preparation instructions here: http://bioconductor.org/packages/devel/bioc/html/PGA.html" />
+                </when>
+                <when value="4">   
+                    <param label="BED File" name="input" format="bed" type="data" argument="-i" help="" />
+                    <param label="Annotation Files" name="anno" type="data_collection" argument="-anno" help="Annotation files for the BED file. Please follow preparation instructions here: http://bioconductor.org/packages/devel/bioc/html/PGA.html" />
+                </when>
+                <when value="5">   
+                    <param label="GTF File" name="input" format="gtf" type="data" argument="-i" help="" />
+                    <param label="Annotation Files" name="anno" type="data_collection" argument="-anno" help="Annotation files for the GTF file. Please follow preparation instructions here: http://bioconductor.org/packages/devel/bioc/html/PGA.html" />
+                </when>
+            </conditional>
+            <param label="Database File" name="db_file" type="data" format="fasta" argument="-db" help="Fasta format database file" />
+            <param label="Spectrum File" name="spectrum_file" type="data" format="mgf" argument="-ms" help="Spectrum file used for identification, mgf format" />
+        </section>
+        <section name="modifications" title="Modifications" expanded="true">
+             <param name="fixed_mod" label="Fixed modification(s)" type="select" multiple="true" argument="-fixMod" help="Fixed modification">
+                <option value="107" selected="true">Oxidation of M (15.99491461956)[0]</option>
+                <option value="6">Carbamidomethylation of C (57.02146372057)[0]</option>
+                <expand macro="modifications" />
+             </param>
+             <param name="var_mod" label="Variable modification(s)" type="select" multiple="true" argument="-varMod" help="Variable modification" >
+                <option value="107">Oxidation of M (15.99491461956)[0]</option>
+                <option value="6" selected="true">Carbamidomethylation of C (57.02146372057)[0]</option>
+                <expand macro="modifications" />
+             </param>
+             <param label="Max Modifications" name="max_mods" type="integer" value="3" argument="-maxVar" help="Max number of variable modifications" />
+             <param label="Unmodified?" name="unmodified" type="boolean" value="false" argument="-um" help="Validation with unrestricted modification searching" />
+             <param label="Consider amino acid substitution modifications?" name="aa" type="boolean" value="false" argument="-aa" help="Whether or not to consider aa substitution modifications when perform modification filtering." />
```

Comment:
booleans only have truevalue and falsevalue.

If you set `truevalue="-aa"` and `falsevalue=""` than your cheetah becomes simply `$modifications.aa`. Which evaluates to `-aa` in the true case and ` ` in the false case :)

Reply:
Ah. I like that solution. Thanks.